benchmark/convolution/opencv.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import absolute_import, print_function, division
 import sys, timeit
-import numpy
+import numpy as np
```

Comment:
this will request change to the string in this file as the string contain code that will be executed.

This file isn't tested, this is why you didn't found this problem with the test.